src/ga4gh/vrs/normalize.py

```diff
@@ -30,9 +42,13 @@ def _normalize_allele(allele, data_proxy):
                                            alleles=alleles,
                                            mode=NormalizationMode.EXPAND,
                                            anchor_length=0)
-        new_allele.location.interval.start = new_ival[0]
-        new_allele.location.interval.end = new_ival[1]
-        new_allele.state.sequence = new_alleles[1]
+        try:
+            new_allele.location.interval.start = new_ival[0]
+            new_allele.location.interval.end = new_ival[1]
+            new_allele.state.sequence = new_alleles[1]
```

Comment:
In the above, sequence from an RSE will incorrectly attempt to update `state.sequence`.

I think section would be made clearer by handling the states with sequence and RSE cases completely separately.

Is there a test for the RSE case?

Reply:
Hi @reece , 

I have pushed my changes for separating out interval.type and state.type. 

If RSE is RepeatedSequenceExpression, I did make a test case for this. [Here](https://github.com/GenomicMedLab/vrs-python/blob/vrs-1.2.0/tests/test_vrs_normalize.py#L92) is the allele that has a RSE as the state. 

Please let me know if these changes addressed your suggestions or if I should make additional changes.

Reply:
Yep, great! Thanks for rounding out support for these cases.